### PR TITLE
replacing toolchain environment variables with relative paths

### PR DESCRIPTION
**Pull request by nidalf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,10 +69,15 @@ WD-Firmware
 ```
 
 ### Additional downloads
-- From the repo unzip riscv-gnu-toolchain-debian.tar.gz to your designated directory for the tool chain
-- From the repo unzip eclipse_mcu_2019_01.7z to your designated directory for the Eclipse MCU
+- Create a new folder ***bin*** inside ***toolchains*** folder in the repository
```
Comment (oferShinaar) [inline README.md]: create empty folder

File: WD-Firmware/demo/build/utils.py [hunk 2]
```diff
@@ -148,26 +148,32 @@ def fnSetOutputFileNames(prefix = ""):
 # set toolchain path
 def fnSetToolchainPath(strTCName, env):
     if strTCName == STR_TC_LLVM:
-       env['RISCV_LLVM_TC_PATH'] = os.getenv(STR_RV_LLVM_ENV_VAR_NAME)
-       # check if the TC environment variable is set or empty
-       if not env['RISCV_LLVM_TC_PATH']:
-         print ("Error: Set environment variable '" + STR_RV_LLVM_ENV_VAR_NAME + "' to point to the RISCV llvm root")
+       env['RISCV_LLVM_TC_PATH'] = os.path.join(os.getcwd(), STR_TOOLCHAIN_BIN, STR_TC_LLVM)
```
Comment (oferShinaar) [inline WD-Firmware/demo/build/utils.py]: to be changed

File: README.md (hunk 1 above)
Comment (nidalf) [reply README.md]: no need, the last changes cover it

File: WD-Firmware/demo/build/utils.py (hunk 2 above)
Comment (nidalf) [reply WD-Firmware/demo/build/utils.py]: done

**Review by oferShinaar — APPROVED** [review]

**Review by ronen-haen-sndk — APPROVED** [review]

**Pull request merged**